### PR TITLE
Add shortcuts for `PATCH` (again)`

### PR DESCRIPTION
**Pull request by homeworkprod**
See #429 for motivation/discussion.

### CODE REVIEW

**Comment by algesten** [comment]
@jsha what do you think about this one?

File: README.tpl [hunk 1]
```diff
@@ -9,6 +9,7 @@
 [std::io::Read]: https://doc.rust-lang.org/stable/std/io/trait.Read.html
 [Agent]: https://docs.rs/ureq/latest/ureq/struct.Agent.html
 [get()]: https://docs.rs/ureq/latest/ureq/fn.get.html
+[patch()]: https://docs.rs/ureq/latest/ureq/fn.patch.html
```
Comment (jsha) [inline README.tpl:12]: This line is not needed. The other entries in this list are things that are linkified in the lib.rs doccomment and are normally provided by rustdoc's intra-doc comments, but need to be copied here for when they are rendered by GitHub Markdown.

**Review by jsha — CHANGES_REQUESTED** [review]
I'm good with this. One small tweak.

File: README.tpl (hunk 1 above)
Comment (homeworkprod) [reply README.tpl:12]: Ok, will remove and force-push.

Comment (homeworkprod) [reply README.tpl]: Done.

**Review by jsha — APPROVED** [review]

**Pull request merged**